plugins/generator-1.21.1/neoforge-1.21.1/mappings/blockstateproperties.yaml

```diff
@@ -0,0 +1,340 @@
+ATTACHED:
+  - BlockStateProperties.ATTACHED
+  - attached
+BERRIES:
+  - BlockStateProperties.BERRIES
+  - berries
+BLOOM:
+  - BlockStateProperties.BLOOM
+  - bloom
+BOTTOM:
+  - BlockStateProperties.BOTTOM
+  - bottom
+CAN_SUMMON:
+  - BlockStateProperties.CAN_SUMMON
+  - can_summon
+CONDITIONAL:
+  - BlockStateProperties.CONDITIONAL
+  - conditional
+CRACKED:
+  - BlockStateProperties.CRACKED
+  - cracked
+CRAFTING:
+  - BlockStateProperties.CRAFTING
+  - crafting
+DISARMED:
+  - BlockStateProperties.DISARMED
+  - disarmed
+DRAG:
+  - BlockStateProperties.DRAG
+  - drag
+DUSTED:
+  - BlockStateProperties.DUSTED
+  - dusted
+ENABLED:
+  - BlockStateProperties.ENABLED
+  - enabled
+EXTENDED:
+  - BlockStateProperties.EXTENDED
+  - extended
+EYE:
+  - BlockStateProperties.EYE
+  - eye
+FALLING:
+  - BlockStateProperties.FALLING
+  - falling
+HANGING:
+  - BlockStateProperties.HANGING
+  - hanging
+HAS_BOOK:
+  - BlockStateProperties.HAS_BOOK
+  - has_book
+HAS_RECORD:
+  - BlockStateProperties.HAS_RECORD
+  - has_record
+INVERTED:
+  - BlockStateProperties.INVERTED
+  - inverted
+IN_WALL:
+  - BlockStateProperties.IN_WALL
+  - in_wall
+LIT:
+  - BlockStateProperties.LIT
+  - lit
+LOCKED:
+  - BlockStateProperties.LOCKED
+  - locked
+OCCUPIED:
+  - BlockStateProperties.OCCUPIED
+  - occupied
+OPEN:
+  - BlockStateProperties.OPEN
+  - open
+PERSISTENT:
+  - BlockStateProperties.PERSISTENT
+  - persistent
+POWERED:
+  - BlockStateProperties.POWERED
+  - powered
+SHORT:
+  - BlockStateProperties.SHORT
+  - short
+SHRIEKING:
+  - BlockStateProperties.SHRIEKING
+  - shrieking
+SIGNAL_FIRE:
+  - BlockStateProperties.SIGNAL_FIRE
+  - signal_fire
+SNOWY:
+  - BlockStateProperties.SNOWY
+  - snowy
+TRIGGERED:
+  - BlockStateProperties.TRIGGERED
+  - triggered
+UNSTABLE:
+  - BlockStateProperties.UNSTABLE
+  - unstable
+VINE_END:
+  - BlockStateProperties.VINE_END

```

Comment:
There is no such thing to be found in BlockStateProperties as VINE_END to my knowledge.  Was this list tested?

Reply:
It was probably defined in earlier versions.